newbius/modules/slurm/variables.tf

```diff
@@ -87,6 +87,10 @@ variable "filestores" {
       device         = string
       mount_path     = string
     }))
+    accounting_storage = optional(object({
```

Comment:
I think `_storage` is redundant suffix here, as it's already placed inside `filestores` variable